src/otp-input/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ButtonProps } from "src/button";
+
+export interface OtpInputProps extends React.AriaAttributes {
+    value?: string | undefined;
+    error?: boolean | undefined;
+    errorMessage?: string | undefined;
+    "data-testid"?: string | undefined;
+    cooldownDuration: number | undefined;
```

Comment:
Mandatory prop should not accept `undefined`. Please review all the other props too.